dashboard/lib/services/csta_enrollment.rb

```diff
@@ -44,12 +44,12 @@ def self.submit(first_name:, last_name:, email:, school_district_name:,
         "submission[15_first]" => first_name,
         "submission[15_last]" => last_name,
         "submission[16]" => email,
-        "submission[5]" => school_district_name,
-        "submission[18]" => school_name,
-        "submission[17_st1]" => street_1,
-        "submission[17_st2]" => street_2,
-        "submission[17_city]" => city,
-        "submission[17_state]" => state,
+        "submission[5]" => school_district_name.titleize,
+        "submission[18]" => school_name.titleize,
+        "submission[17_st1]" => titleize_address(street_1),
+        "submission[17_st2]" => titleize_address(street_2),
+        "submission[17_city]" => city.titleize,
+        "submission[17_state]" => get_us_state_abbr(state),
```

Comment:
I added the state name to abbreviation conversion in the AFE controller in [this PR](https://github.com/code-dot-org/code-dot-org/pull/35707/files) (sorry, late addition -- didn't realize you would have it in your PR), so I think we can remove it here?

Reply:
Ah, nvm! This is your own custom version that can handle two letter state codes. Proceed!

Reply:
I'd like us to consider adopting my approach here for two reasons:

1. The address format AFE and CSTA are expecting, respectively, feels like it should be a concern of these classes that exist to shape params into the appropriate format and post to external services - I'd like AmazonFutureEngineerController to only worry about where we get this information, not how we format it.
2. The `get_us_state_abbr_from_name` method I recommended in that PR has a weakness: If you pass it input which is _already_ a state abbreviation and does not need to be modified, it will return `nil`.  I'd like the API to be more flexible than that, even if we expect to always get full state names now that we're using a dropdown; maybe something in our NCES data isn't formatted like we expect?  That's why I've introduced the `get_us_state_abbr` helper in this PR, which accepts name or abbreviation and standardizes them.

If you'll merge https://github.com/code-dot-org/code-dot-org/pull/35707 first, I'll be happy to modify this PR to update all the state formatting logic accordingly.

Reply:
Reasonable on both points -- #35707 is merged, I think updating this PR to do state formatting in the services makes sense if you want to make that change here!